lib/setupmanagers/qemuhck/qemu_machine.rb

```diff
@@ -193,6 +193,13 @@ def options_replacement_list
       }
     end
 
+    def double_memory
```

Comment:
Seems confusing name for function and variable. 
Maybe: memory_for_virtio_mem or pluggable_memory?

Reply:
No, this is just double memory. We run QEMU with option -m memory,maxmemory=double_memory. I hope that this reserve will be enough. While memory_for_virtio_mem is 1G in our configuration and pluggable_memory is the same as a run memory.